lib/counter.js

```diff
@@ -217,7 +267,8 @@ TimestampedCounter.prototype.incrby = function(amount, eventObj, callback) {
   if (eventObj) eventObj = String(eventObj);
   var deferred = Q.defer();
   var cb = utils.createRedisCallback(deferred, callback);
-  this._incrby(amount, eventObj, cb);
+  // -- null is passed to mean now() for timestamp
+  this._incrby(amount, null, eventObj, cb);
```

Comment:
You could also shorten this entire function's body to just:
`return this.incrbyWithDate(amount, null, eventObj, cb);`